.travis.yml

```diff
@@ -1,5 +1,11 @@
 ---
-language: python # need for pip
+language: python
+
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
```

Comment:
In order to pinpoint a particular version of python and all related tools (e.g. pip, python-openstack, etc...)